README.md

```diff
@@ -0,0 +1,494 @@
+    * `message` [`<String>`][] - Static message: `'Successfully added meta property'`
+    * `key` [`<String>`][] - The added key name
+    * `value` [`<String>`][] | [`<Number>`][] | [`<Boolean>`][] | [`<Object>`][] | [`<Array>`][] - The value
+
+Emitted when a meta property is successfully added.  This meta property will be attached to each log message.
+
+### Event: `'cleared'`
+* [`<Object>`][]
+    * `message` [`<String>`][] - A message indicating that everything was sent, or that nothing needed to be sent
+
+When all log lines have been sent to LogDNA, this event is emitted.  If it emits after lines have successfully been sent,
+then the message will be `'All accumulated log entries have been sent'`.  If there were no lines to be sent
+(for example, if `flush()` was called proactively), then the message will be `'All buffers clear; Nothing to send'`.
+
+
+### Event: `'error'`
+* [`<TypeError>`][] | [`<Error>`][]
+    * `message` [`<String>`][] - The error message
+    * `meta` [`<String>`][] - Meta object populated with different information depending on the error
+
+This is emitted when an asynchronous error is encountered.  Depending on the context, `meta` will contain
+different pieces of information about the error.
+
+**Error while sending to LogDNA**
+
+The meta data for an error encountered during an HTTP call to LogDNA will have the following `meta` property in the error:
+
+* `actual` [`<String>`][] - The raw error message from the HTTP client
+* `code` [`<String>`][] | [`<Number>`][] - The HTTP agent's "code", or `statusCode` value
+* `firstLine` [`<String>`][] - The first log line in the buffer that was sending
+* `lastLine` [`<String>`][] - The last log line in the buffer that was sending
+* `retrying` [`<Boolean>`][] - Whether or not an attempt will be made to resend the payload
+* `attempts` [`<Number>`][] - The number of consecutive failures
+
+**Error while calling `log()`**
+
+When `log()` is called directly, or indirectly (via a convenience method), errors can be emitted if certain validations fail.  
+If an invalid log level is provided, or if a bad data type is found for the `options` parameter, the `meta` property of the error
+will contain the following:
+
+* `got` [`<String>`][] - Description of the invalid input.  Will depend on error context.
+* `expected` [`<String>`][] - The allowable log levels, if `options` is an invalid log level
+* `used` [`<String>`][] - If a bad `level` is used in `options`, it will be ignored, and the default will be used.  This indicates what that value is.
+
+
+### Event: `'removeMetaProperty'`
+* [`<Object>`][]
+    * `message` [`<String>`][] - Static message: `'Successfully removed meta property'`
+    * `key` [`<String>`][] - The key that was removed
+
+This is emitted when a key (and implied value) are removed from the global `meta` object.  If the key does not exist,
+then a [`warn`](#event-warn) event with the same signature will be emitted instead.
+
+
+### Event: `'send'`
```

Comment:
again, `sent` or `send`?

Reply:
The event should match the function for ease.  I think ease of use trumps grammar in these cases.